vaadin-lumo-theme-flow-parent/vaadin-lumo-theme-flow/src/main/java/com/vaadin/flow/theme/lumo/LumoIcon.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2000-2022 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.vaadin.flow.theme.lumo;
+
+import com.vaadin.flow.component.icon.Icon;
+import com.vaadin.flow.component.icon.IconFactory;
+
+import java.util.Locale;
+
+/**
+ * Enumeration of all icons in the
+ * <a href= "https://vaadin.com/docs/latest/ds/foundation/icons">Lumo Icons</a>
```

Comment:
Let's use the new docs link:

```suggestion
 * <a href= "https://vaadin.com/docs/latest/components/icons">Lumo Icons</a>
```

Reply:
Done